### PR TITLE
fix: readme with install instructions and dependency chart repo

### PR DESCRIPTION
**Pull request by santinoncs**

### CODE REVIEW

**Comment by rccrdpccl** [comment]
I think it would be best to package the whole thing, so that this can be installed without downloading the source code

**Comment by santinoncs** [comment]
> I think it would be best to package the whole thing, so that this can be installed without downloading the source code

The instructions actually points to this direction also. Install without downloading the source code.

File: Chart.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: go-ratelimit
 description: A Ratelimit Service Helm chart for Kubernetes
 type: application
-version: 1.0.2
+version: 1.0.3
 appVersion: 1.5.0
```
Comment (mehyedes) [inline Chart.yaml:6]: Hey @santinoncs 
Linking to my [comment](https://github.com/softonic/go-ratelimit-chart/issues/6#issuecomment-845848638), it seems that version `1.5.0` is not available as a docker image. So maybe it makes sense to change the default image tag in the values file.

File: Chart.yaml [hunk 2]
```diff
@@ -2,10 +2,10 @@ apiVersion: v2
...
 dependencies:
 - name: "redis-sharded"
   version: 0.1.0
   alias: "redish"
-  repository: "@softonic-public"
+  repository: "https://charts.softonic.io"
```
Comment (mehyedes) [inline Chart.yaml:11]:
```suggestion
  repository: "https://charts.softonic.io"
  condition: redish.enabled
```

File: Chart.yaml [hunk 3]
```diff
@@ -2,10 +2,10 @@ apiVersion: v2
...
-version: 1.0.2
+version: 1.0.3
 appVersion: 1.5.0
 dependencies:
 - name: "redis-sharded"
-  version: 0.1.0
+  version: 0.3.0
   alias: "redish"
-  repository: "@softonic-public"
+  repository: "https://charts.softonic.io"
```
Comment (mehyedes) [inline Chart.yaml:11]: What do you think of adding a condition here to make it possible to disable the installation of the redis chart from the values file?
```suggestion
  repository: "https://charts.softonic.io"
  condition: redish.enabled
```

File: README.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ For more insights of envoy rate limit architecture, [Global rate limit Envoy](ht
 # Quick Start
 
 ```bash
-helm repo add softonic https://charts.softonic.io
-helm install redis-sharded softonic/go-ratelimit-chart
+helm dep up
```
Comment (mehyedes) [inline README.md]: Since the redis chart is now packaged with the `go-ratelimit`, is this step still needed?

**Review by mehyedes — APPROVED** [review]
LGTM. 👍 
**PS:** Thank you very much for the quick fix @santinoncs 🏅

**Pull request merged**